test/cri-metrics.bats

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env bats
+# vim: set syntax=sh:
+
+load helpers
+
+function setup() {
+	if [[ "$CONTAINER_DEFAULT_RUNTIME" == "containerd-shim-kata-v2" ]]; then
+		# the tests read the cgroup files directly, which is not possible with containerd-shim-kata-v2; skip for now.
+		skip
+	fi
```

Comment:
We also skip tests for Kata via:

- [cri-o/cri-o/contrib/test/ci/vars.yml](https://github.com/cri-o/cri-o/blob/main/contrib/test/ci/vars.yml)

Would that work for you?

Reply:
yeah, I guess this is what I was trying to do. Thank you!